examples/04_New Models.ipynb

```diff
@@ -690,6 +690,140 @@
+   "metadata": {},
+   "source": [
+    "Now let's test out this capability. To do so, we'll use the `time` package to compare the direct model to our original timeseries model. \n",
+    "\n",
+    "Let's start by creating an instance of our timeseries model, calculating the time of event, and timing this computation. Note that for a state transition model, `time_of_event` still returns the time at which `threshold_met` returns true for each event, but this is calculated by simulating to threshold."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import time \n",
+    "\n",
+    "m_timeseries = ThrownObject_ST()\n",
+    "x = m_timeseries.initialize()\n",
+    "print(m_timeseries.__class__.__name__, \"(Direct Model)\" if m_timeseries.is_direct else \"(Timeseries Model)\")\n",
+    "tic = time.perf_counter()\n",
+    "print('Time of event: ', m_timeseries.time_of_event(x, dt = 0.05))\n",
+    "toc = time.perf_counter()\n",
+    "print(f'execution: {(toc-tic)*1000:0.4f} milliseconds')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's do the same using our direct model implementation. In this case, when `time_to_event` is called, the event time will be estimated directly from the state, instead of through simulation to threshold. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "m_direct = DirectThrownObject()\n",
+    "x = m_direct.initialize()  # Using Initial state\n",
+    "# Now instead of simulating to threshold, we can estimate it directly from the state, like so\n",
+    "print('\\n', m_direct.__class__.__name__, \"(Direct Model)\" if m_direct.is_direct else \"(Timeseries Model)\")\n",
+    "tic = time.perf_counter()\n",
+    "print('Time of event: ', m_direct.time_of_event(x))\n",
+    "toc = time.perf_counter()\n",
+    "print(f'execution: {(toc-tic)*1000:0.4f} milliseconds')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Notice that execution is significantly faster for the direct model. This will be even more pronounced for events that occur later in the simulation. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example, we have illustrated how to use a direct model. Direct models are a powerful tool for estimating the time of an event directly from the system state. By avoiding the process of state transitions, direct models can provide more efficient event time estimates. Additionally, the direct model approach is not limited to physics-based models. It can also be applied to data-driven models that can map sensor data directly to the time of an event. \n",
+    "\n",
+    "In conclusion, direct models offer an efficient and versatile approach for prognostics modeling, enabling faster and more direct estimations of event times. "
```

Comment:
We might want to acknowledge that this is a very simple example, as there are no inputs. For models with inputs, future loading must be provided to time_of_event (see 01_Simulation > Future Loading).

It also might be worth mentioning that in those cases, most direct models will somehow encode or discretize the future loading profile to use it in a direct estimation of ToE.